patches/minecraft/net/minecraft/block/BlockCactus.java.patch

```diff
@@ -9,7 +9,26 @@
  {
      public static final PropertyInteger field_176587_a = PropertyInteger.func_177719_a("age", 0, 15);
      protected static final AxisAlignedBB field_185593_b = new AxisAlignedBB(0.0625D, 0.0D, 0.0625D, 0.9375D, 0.9375D, 0.9375D);
-@@ -110,8 +110,8 @@
+@@ -48,7 +48,8 @@
+             if (i < 3)
+             {
+                 int j = ((Integer)p_180650_3_.func_177229_b(field_176587_a)).intValue();
+-
```

Comment:
@LexManos My point was that I was _attempting not to mess with the whitespace_ and the how to contribute page does not make it clear _how to do that_ so I made a best guess. If I did it wrong, that's fine, I'll fix it, but the instructions on what I did wrong don't exist.  I can fix this here and now, but that doesn't help someone else who's new.  Prior to starting this endeavor _I didn't know how to read patch files._  I'm slowly starting to understand what it is that you want, but please, rather than being snarky all you have to say is "this added that extra line that makes the patch files harder to read. It's cleaner if you only insert lines, even if there's already a blank one."